cura/API/__init__.py

```diff
@@ -1,17 +1,20 @@
 # Copyright (c) 2018 Ultimaker B.V.
 # Cura is released under the terms of the LGPLv3 or higher.
+from PyQt5.QtCore import QObject, pyqtProperty
+
 from UM.PluginRegistry import PluginRegistry
 from cura.API.Backups import Backups
 from cura.API.Interface import Interface
+from cura.API.Account import Account
+
 
 ##  The official Cura API that plug-ins can use to interact with Cura.
 #
 #   Python does not technically prevent talking to other classes as well, but
 #   this API provides a version-safe interface with proper deprecation warnings
 #   etc. Usage of any other methods than the ones provided in this API can cause
 #   plug-ins to be unstable.
-
-class CuraAPI:
+class CuraAPI(QObject):
```

Comment:
It's either a pyqtProperty or a pyqtSlot. You want to access this in QML.

Reply:
Ah, right. But Cura in itself has a dependency on Qt in the first place. I do agree that we need to try and keep them as seperate as possible (eg; we should have done this for the settings). In this case it would introduce a fair bit of code duplication.